modules/openapi-generator/src/main/resources/kotlin-spring/libraries/spring-boot/README.mustache

```diff
@@ -0,0 +1,21 @@
+# {{#title}}{{title}}{{/title}}{{^title}}Generated Kotlin Spring Boot App{{/title}}
+
+This Kotlin based [Spring Boot](https://spring.io/projects/spring-boot) application has been generated using the [OpenAPI Generator](https://github.com/OpenAPITools/openapi-generator).
+
+## Getting Started
+
+This document assumes you have either maven or gradle available, either via the wrapper or otherwise. This does not come with a gradle / maven wrapper checked in.
+
+By default a [`pom.xml`](pom.xml) file will be generated. If you specified `gradleBuildFile=true` when generating this project, a `build.gradle.kts` will also be generated. Note this uses [Gradle Kotlin DSL](https://github.com/gradle/kotlin-dsl).
+
+To build the project using maven, run:
+```bash
+mvn package && java -jar target/openapi-spring-1.0.0.jar
```

Comment:
`openapi-spring` should be replaced with `{{artifactId}}` here and a couple lines below (the version possibly as well).